app/routes.py

```diff
@@ -0,0 +1,66 @@
+import os.path
+# using werkzeug.exceptions in order to get rid of pyramid
+# from pyramid.httpexceptions import HTTPBadRequest and other flask stuff
+# from pyramid.view import view_config
+# from pyramid.response import Response
+from flask import Response
+import six  # for compatibility Python 2 / 3
+from PIL import Image
+from app import app
+from app.helpers import check_color_channels
+from app.helpers import check_path
+
+# Python2/3 compatibility
+if six.PY3:
+    from io import BytesIO
+else:
+    from StringIO import StringIO
```

Comment:
We support only py3 for the new services, so all stuff related to py2 should be removed (six et. al)

Reply:
Aggree with this, it make the code less readable.

Reply:
support of py2 deleted